archive_verify/workers.py

```diff
@@ -93,48 +30,62 @@ def compare_md5sum(archive_dir):
     else: 
         return True
 
-def verify_archive(archive, archive_path, description, config):
+
+def get_pdc_client_class(config):
     """
-    Our main worker function. This will be put into the RQ/Redis queue when the /verify endpoint gets called. 
-    Downloads the specified archive from PDC and then verifies the MD5 sums. 
+    Determines which PDC Client should be used.
 
-    :param archive: The name of the archive we shall download
-    :param archive_path: The path to the archive on PDC
-    :param description: The unique description that was used when uploading the archive to PDC
     :param config: A dict containing the apps configuration
-    :returns A JSON with the result that will be kept in the Redis queue
+    :returns A PDC Client.
     """
-    dest_root = config["verify_root_dir"]
-    dsmc_log_dir = config["dsmc_log_dir"]
-    whitelist = config["whitelisted_warnings"]
+    return MockPdcClient if config.get("pdc_client", "PdcClient") == "MockPdcClient" else PdcClient
```

Comment:
I'm not sure if this needs to have a default value, but that is probably just a mater of taste.

Reply:
Probably overkill, but I like keeping it there for clarity.